e2e/guardrails/basal_rate_schedules.js

```diff
@@ -0,0 +1,61 @@
+const { limits } = require('../../src/index');
+
+var basalRateSchedules = (test) => {
+    it('can set max limit', async () => {
+        await test.settingsScreen.SetBasalRates(
+            [{ time: '12:00 AM', unitsPerHour: limits.basalRates.max.limit }],
+            { fromSettings: true, toSettings: false }
```

Comment:
What is the "fromSettings" and "toSettings"?

Reply:
removed. Now with the returning of sub screens I can do away with all this